src/main/java/lottogame/domain/lotto/LottoNumber.java

```diff
@@ -1,93 +1,64 @@
 package lottogame.domain.lotto;
 
-import lottogame.utils.DuplicateLottoNumberException;
-import lottogame.utils.InvalidBonusBallNumberException;
 import lottogame.utils.InvalidLottoNumberRangeException;
 
-import java.util.ArrayList;
-import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 public class LottoNumber {
-    private static final int LOTTO_MIN = 1;
-    private static final int LOTTO_MAX = 45;
-    private static final int LOTTO_NUMBER_VOLUME = 6;
-    private final List<Integer> numbers = new ArrayList<>();
-
-    public LottoNumber() {
-        for (int i = 0; i < LOTTO_NUMBER_VOLUME; i++) {
-            numbers.add(makeNumber());
+    public static final int LOTTO_NUMBER_MIN = 1;
+    public static final int LOTTO_NUMBER_MAX = 45;
+    private final int number;
+
+    private static final Map<Integer, LottoNumber> lottoNumbers = new HashMap<>();
+    static {
+        for (int i = LOTTO_NUMBER_MIN; i <= LOTTO_NUMBER_MAX; i++) {
+            lottoNumbers.put(i, new LottoNumber(i));
         }
-        sortNumbers();
-    }
-
-    public LottoNumber(List<Integer> numbers) {
-        validate(numbers);
-        this.numbers.addAll(numbers);
-        sortNumbers();
     }
 
-    public LottoNumber(int bonusBall) {
-        if (invalidRange(bonusBall)) {
-            throw new InvalidBonusBallNumberException();
-        }
-        numbers.add(bonusBall);
+    public LottoNumber(int number) {
+        this.number = number;
     }
 
-    private Integer makeNumber() {
-        int number = LottoNumberGenerator.generate(LOTTO_MIN, LOTTO_MAX);
-        if (numbers.contains(number)) {
-            return makeNumber();
+    public static LottoNumber valueOf(int number) {
+        if (!lottoNumbers.containsKey(number)) {
+            throw new InvalidLottoNumberRangeException();
         }
-        return number;
-    }
-
-    private void validate(List<Integer> numbers) {
-        validNumberRange(numbers);
-        duplicateNumbers(numbers);
+        return lottoNumbers.get(number);
     }
 
-    private void sortNumbers() {
-        Collections.sort(numbers);
-    }
-
-    private void validNumberRange(List<Integer> numbers) {
-        numbers.stream()
-                .filter(number -> invalidRange(number))
-                .findAny()
-                .ifPresent(s -> {
-                    throw new InvalidLottoNumberRangeException();
-                });
+    public static List<LottoNumber> of(List<Integer> numbers) {
+        return numbers.stream()
+                .map(number -> LottoNumber.valueOf(number))
```

Comment:
```suggestion
                .map(LottoNumber::valueOf)
```
요 친구는 요렇게 메서드 참조로 간단하게도 가능해요